docs/preparation.rst

```diff
@@ -151,10 +151,16 @@ The ``dicomsort`` command-line tool is a utility to move your flat- or DICOMDIR-
                             folder name ("SeriesDescription" and "ProtocolName"
                             are both used as fallback) (default:
                             SeriesDescription)
-      -r, --rename          Flag to rename the DICOM files to a PatientName_Series
-                            Number_SeriesDescription_AcquisitionNumber_InstanceNum
-                            ber scheme (recommended for DICOMDIR data) (default:
-                            False)
+      -r, --rename          Flag to rename the DICOM files (recommended for
+                            DICOMDIR data) (default: False)
+      -s, --scheme          The naming scheme for renaming. Follows the Python string
```

Comment:
I'm not sure if it's a good idea. You would have to handle three cases then:
1) rename == False -> do not rename
2) rename == True -> rename using the standard scheme
3) rename is str -> rename using the user-provided scheme given in the string
This would make handling schemes with errors more tricky as a string would resolve to True even if it's not a correct scheme. What if the user enters "1" - is it a wrong scheme or a correct True value? Plus, having an explicit scheme argument makes it easier to document the default value.